### DISCUSSION TITLE
SAAS

### DISCUSSION THREAD

**Discussion by PeikLiOng** [datalust/seq-tickets#2022] [Help]
Hello,
Can SeqLog works with SAAS type of application?

Thank you,
Ong

**Comment by liammclennan** [comment]
Hi @PeikLiOng 

Seq's API is HTTP, so it works with any client that can generate HTTP requests.

**Comment by PeikLiOng** [comment]
Hi Liam,

I have a new application which is SAAS based how should I install SeqLog (I have purchased the DataCenter version)?

Thanks,
Ong Peik Li
Head - Hotel
E:  ***@***.******@***.***>
D:  +603 2333 3373
[https://www.rwgenting.com/content/dam/approved/rw-genting/web/outlook-signatures/RWG-email-logo-204x49.png]<https://www.rwgenting.com/>

**Comment by liammclennan** [comment]
Hi @PeikLiOng ,

If you have a windows server then https://docs.datalust.co/docs/getting-started and to install on linux https://docs.datalust.co/docs/getting-started-with-docker.

**Comment by PeikLiOng** [comment]
Hi Liam,

My SAAS application will be on a platform that will not be owned and managed by us however I would like to use Seq Log on some of the log files generated from the SAAS application. How do I go about this? We have implemented Seq Logs but for on-prem applications and is first time for SAAS. Appreciate your guidance on this.

Thanks,
Ong Peik Li
Head - Hotel
E:  ***@***.******@***.***>
D:  +603 2333 3373
[https://www.rwgenting.com/content/dam/approved/rw-genting/web/outlook-signatures/RWG-email-logo-204x49.png]<https://www.rwgenting.com/>

**Reply by liammclennan** [reply]
Seqcli (https://github.com/datalust/seqcli#ingest) can be used to ingest log files into Seq. Because the files will not be in the correct format some transformation will be required. If you would like to send a redacted example log file to support@datalust.co we can have a go at importing it. 

Another option is to use fluentbit as in [this example](https://blog.datalust.co/tailing-a-log-file-or-folder-with-fluent-bit-and-seq/).
